app/app.rb

```diff
@@ -0,0 +1,27 @@
+require 'sinatra/base'
+require_relative 'data_mapper_setup'
+require 'sinatra/flash'
+require 'sinatra/partial'
+require_relative './helpers'
+
+class Chitter < Sinatra::Base
+  enable :sessions, :method_override
+  set :session_secret, 'super secret'
+  register Sinatra::Flash
+  register Sinatra::Partial
+
+  require_relative './models/send_tag_alert'
+  require_relative 'controllers/login'
```

Comment:
It's great that you've broken things out into controllers; I'm not sure you need quite so many controllers however. I think login/logout/signup are all related to auth & accounts, so I'd be tempted to put those in a single AccountController, while peeping and replying are both related to Peeps, so I would put those in a PeepsController. Controllers in general should be organised around concerns, rather than individual routes.